### PR TITLE
feat: introducing Amazon Web Service (AWS)

### PR DESCRIPTION
**Pull request by NicolasFujisawa**

### CODE REVIEW

**Review by jasondavindev — APPROVED** [review]
Fix the personal sentences (you)

File: infrastructure/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+The gain of the flexibility in selecting the O.S, programming language, a lot of variation in solutions. Easy for newcomers with good usability and diversity to work in their, by the console, CLI and SDK's. Most economic platform to host applications in the market and plans to pay as you go only. And a secure end-to-end to protect and fortify the applications.
+
+## Benefits
+
+- Easy to use
+- Economic
+- Flexible
+- Trusted
+- Auto-scaling and performance
+- Secure
```
Comment (toledompm) [inline infrastructure/readme.md:16]: Should we add reliability to the list of benefits?

Comment (NicolasFujisawa) [reply infrastructure/readme.md:16]: Easy to use is an approach to reliability?

Comment (toledompm) [reply infrastructure/readme.md:16]: Its more to do with their approach and tools for fault-tolerance:
https://wa.aws.amazon.com/wellarchitected/2020-07-02T19-33-23/wat.pillar.reliability.en.html

**Review by toledompm — APPROVED** [review]

**Pull request merged**